src/components/popover/popover.ts

```diff
@@ -99,3 +99,14 @@ export class Popover extends Component<Props, SpreadsheetChildEnv> {
     );
   }
 }
+
+Popover.props = {
+  position: Object,
+  marginTop: { type: Number, optional: true },
```

Comment:
interface says it's mandatory, same goes for flipHorizontalOffset and flipVerticalOffset